GateWatcher/aioniq/ingest/parser.yml

```
@@ -0,0 +1,112 @@
+          observer.version: '0.2'
+          observer.hostname: '{{json_load.message.host}}'
+          observer.type: 'firewall'
+          network.transport: '{{json_load.message.proto}}'
+          network.protocol: '{{json_load.message.app_proto}}'
+          gatewatcher.type: '{{json_load.message.type}}'
+          gatewatcher.state: '{{json_load.message.state}}'
+          gatewatcher.gcenter: '{{json_load.message.gcenter}}'
+          gatewatcher.gcap: '{{json_load.message.gcap}}'
+          gatewatcher.flow_id: '{{json_load.message.flow_id}}'
+          gatewatcher.timestamp_analyzed: '{{json_load.message.timestamp_analyzed}}'
+          gatewatcher.timestamp_detected: '{{json_load.message.timestamp_detected}}'
+          gatewatcher.event_type: '{{json_load.message.event_type}}'
+          event.category: ['network']
+          event.type: ['info']
+          event.severity: '{{json_load.message.severity}}'
+      - set:
+          observer.mac: '["{{json_load.message.ether.src_mac}}", "{{json_load.message.ether.dest_mac}}"]'
+        filter: '{{json_load.message.ether != null}}'
+      - name: translate
+        dictionary:
+          alert: alert
+        mapping:
+          gatewatcher.event_type: event.kind
+        fallback: event
+
+  malcore:
+    actions:
+      - set:
+          file.name: '{{json_load.message.fileinfo.filename}}'
+          file.size: '{{json_load.message.fileinfo.size}}'
+          file.hash.md5: '{{json_load.message.fileinfo.md5}}'
+          file.hash.sha256: '{{json_load.message.fileinfo.sha256}}'
+          url.path: '{{json_load.message.http.url}}'
+          url.domain: '{{json_load.message.http.hostname}}'
+          http.version: '{{json_load.message.http.version}}'
+          http.request.method: '{{json_load.message.http.http_method}}'
+          user_agent.original: '{{json_load.message.http.http_user_agent}}'
+          http.response.status_code: '{{json_load.message.http.status}}'
+          gatewatcher.retroact: '{{json_load.message.retroact}}'
+          gatewatcher.nb_rescans: '{{json_load.message.nb_rescans}}'
+          gatewatcher.malcore.code: '{{json_load.message.code|string}}'
+          gatewatcher.malcore.file: '{{json_load.message.file}}'
+          gatewatcher.malcore.magic: '{{json_load.message.magic}}'
+          gatewatcher.malcore.replica: '{{json_load.message.replica}}'
+          gatewatcher.reporting_token: '{{json_load.message.reporting_token}}'
+          gatewatcher.malcore.detail_threat_found: '{{json_load.message.detail_threat_found}}'
+          event.category: ['malware']
+          event.type: ['info']
+
+  suricata:
+    actions:
+      - set:
+          source.bytes: '{{json_load.message.flow.bytes_toclient}}'
+          source.packets: '{{json_load.message.flow.pkts_toclient}}'
+          destination.bytes: '{{json_load.message.flow.bytes_toserver}}'
+          destination.packets: '{{json_load.message.flow.pkts_toserver}}'
```

```suggestion          
          source.bytes: '{{json_load.message.flow.bytes_toserver}}'
          source.packets: '{{json_load.message.flow.pkts_toserver}}'
          destination.bytes: '{{json_load.message.flow.bytes_toclient}}'
          destination.packets: '{{json_load.message.flow.pkts_toclient}}'
```

You might have inverted these fields e.g.:

> [destination.bytes](https://www.elastic.co/guide/en/ecs/current/ecs-destination.html#field-destination-bytes): Bytes sent from the destination to the source.

---

fixed